### DISCUSSION TITLE
is there a way to port animations from fortnite v18.10?

### DISCUSSION THREAD

**Discussion by astaliyaa** [gildor2/UEViewer#242] [Q&A]
when i try to export anims from fortnite in umodel it gives me an error and when i export the montage it doesnt show up in the animations folder

**Comment by gildor2** [comment]
Montages are different assets, plus they're animating scene objects, not skeletal meshes. They are not supported (as well as scenes themselves).

**Reply by astaliyaa** [reply]
oh ok thanks!
